packages/components/src/internal/components/domainproperties/samples/SampleTypePropertiesPanel.tsx

```diff
@@ -338,6 +356,29 @@ class SampleTypePropertiesPanelImpl extends React.PureComponent<
                         </Col>
                     </Row>
                 )}
+
+                {allowTimepointProperties && !appPropertiesOnly && (
+                    <Row className="margin-top">
+                        <Col xs={2}> Auto-Link Data to Study </Col>
```

Comment:
As seen in the `AssayPropertiesInput` case, I think we should add a help tooltip to explain this new option for sample types.

Reply:
Done!